### ISSUE TITLE
Erring with 'RangeError: Line number out of range ...'

### ISSUE THREAD

**Issue by matthewadams** [eslint/eslint#8403]
**Tell us about your environment**
macOS Sierra, latest
Using eslint via `standard@9.0.2`

* **ESLint Version:**
`~3.18.0` (as per `standard@9.0.2`'s `package.json` [line 15](https://github.com/feross/standard/blob/v9.0.2/package.json#L15)

* **Node Version:**
v6.10.1

* **npm Version:**
3.10.10

**What parser (default, Babel-ESLint, etc.) are you using?**
n/a

**Please show your full configuration:**
See https://github.com/feross/standard/tree/v9.0.2

**What did you do? Please include the actual source code causing the issue.**
```js
class Foo {}
// next line has a single blank space
 
```

**What did you expect to happen?**
I expected `eslint` not to throw an error.

**What actually happened? Please include the actual, raw output from ESLint.**
```
$ standard
standard: Unexpected linter output:

RangeError: Line number out of range (line 4 requested, but only 3 lines present).
    at SourceCode.getIndexFromLoc (/Users/ADA6011/tmp/standard/node_modules/eslint/lib/util/source-code.js:389:19)
    at Object.fix (/Users/ADA6011/tmp/standard/node_modules/eslint/lib/rules/no-multiple-empty-lines.js:116:52)
    at RuleContext.report (/Users/ADA6011/tmp/standard/node_modules/eslint/lib/rule-context.js:127:34)
    at Program:exit.allLines.reduce.concat.reduce (/Users/ADA6011/tmp/standard/node_modules/eslint/lib/rules/no-multiple-empty-lines.js:108:37)
    at Array.reduce (native)
    at EventEmitter.Program:exit (/Users/ADA6011/tmp/standard/node_modules/eslint/lib/rules/no-multiple-empty-lines.js:93:22)
    at emitOne (events.js:101:20)
    at EventEmitter.emit (events.js:188:7)
    at NodeEventGenerator.applySelector (/Users/ADA6011/tmp/standard/node_modules/eslint/lib/util/node-event-generator.js:265:26)
    at NodeEventGenerator.applySelectors (/Users/ADA6011/tmp/standard/node_modules/eslint/lib/util/node-event-generator.js:294:22)

If you think this is a bug in `standard`, open an issue: https://github.com/feross/standard/issues
```

Looks like the problem is in `no-multiple-empty-lines.js`.

See related issue in `standard` at https://github.com/feross/standard/issues/833.

**Comment by matthewadams** [comment]
Looks like possible duplicate of https://github.com/eslint/eslint/issues/8401

**Comment by platinumazure** [comment]
@not-an-aardvark Can you please confirm this is a dup of #8401 and close if so? Thanks!

**Comment by matthewadams** [comment]
dup of https://github.com/eslint/eslint/issues/8401

**Comment by honomoa** [comment]
I can reproduce by add only two space at last line in any file.
